### DISCUSSION TITLE
[1987080]深入后室

### DISCUSSION THREAD

**Discussion by SunshnX** [pjy612/SteamManifestCache#67] [Share]
appid:1987080
tg:SunshnP
内测码:6BBDE78577749475
[Inside backrooms.zip](https://github.com/pjy612/SteamManifestCache/files/13694612/Inside.backrooms.zip)
求个内测名额QwQ

**Comment by ZEERDEER** [comment]
#56 有人发过了

**Reply by SunshnX** [reply]
哦，没看到，怎么删除啊
